### PR TITLE
Unify release, tag & image names

### PR DESCRIPTION
**Pull request by KatyTaylor**
update github build script to unify the naming of github releases, github tags and docker images, to make deployments less confusing

### CODE REVIEW

File: .github/workflows/automated_release.yml [hunk 1]
```diff
@@ -61,15 +61,18 @@ jobs:
...
+
+      - name: Set release tag
+        run: echo "RELEASE_TAG=v$RELEASE_NAME" >> $GITHUB_ENV
 
       - name: Create release
         uses: ncipollo/release-action@v1.8.8
         with:
-          name: ${{ env.RELEASE_VERSION }}
-          tag: v${{ env.RELEASE_VERSION }}
+          name: ${{ env.RELEASE_NAME }}
```
Comment (KatyTaylor) [inline .github/workflows/automated_release.yml:74]: So I would give the release name a `v` as well as the tag name, just so there are `v`s everywhere and it's unambiguous if people don't know exactly which string they're meant to use in the Ansible command.

File: crawler/config/defaults.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
 ###
 # Baracoda
 ###
-BARACODA_BASE_URL = f"http://{LOCALHOST}:8000"
+BARACODA_BASE_URL = f"http://{LOCALHOST}:7900"
```
Comment (KatyTaylor) [inline crawler/config/defaults.py:41]: why this change?

Comment (sdjmchattie) [reply crawler/config/defaults.py:41]: Oh yeah sorry.  It's partly because I have to make this change manually every time I hook it all up locally.  If you look just below the fold here you'll see that we're using the same port for Cherrytrack which is clearly never going to work.

File: .github/workflows/automated_release.yml (hunk 1 above)
Comment (sdjmchattie) [reply .github/workflows/automated_release.yml:74]: I think I'd rather keep consistency with non-Docker repos for now.  Thanks for the suggestion though.  If it still presents a problem we can change it.

**Comment by sdjmchattie** [comment]
I'm going to merge this to see if it functions correctly before applying it to other repos.

**Review by sdjmchattie — APPROVED** [review]
